### DISCUSSION TITLE
Calcular la oscuridad de un color en Blogger

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#16] [Announcements]
# Calcular la oscuridad de un color en Blogger

Cómo detectar la oscuridad de un color en Blogger sin javascript

https://www.zkreations.com/2023/01/calcular-oscuridad-color-blogger.html

**Comment by maviyapim** [comment]
When are you going to talk about creating a partition that works with a tag?

**Reply by danieIabel** [reply]
I did not understand the question friend, could you explain it better?

**Reply by maviyapim** [reply]
Creating fields like in bison theme. Example . Creating a label-operated space such as food-travel sections.

**Reply by danieIabel** [reply]
That project is called BloggerJS and it will be available soon, you can see more things we are working on and also share with the community here: https://t.me/zkreation

**Comment by davidlimonche** [comment]
esto es una funcion para el modo oscuro/modo claro?

**Reply by danieIabel** [reply]
No amigo, sirve para detectar la oscuridad de un color de blogger, no tiene nada que ver con javascript
